charts/rancher-monitoring/v0.1.3/values.yaml

```diff
@@ -259,7 +259,7 @@ grafana:
       repository: rancher/prometheus-auth
       tag: v0.2.0
     proxy:
-      repository: rancher/nginx
+      repository: rancher/library-nginx
       tag: 1.19.2-alpine
   nodeSelectors: []
   resources:
```

Comment:
Can be dropped now since https://github.com/rancher/system-charts/pull/341 already addresses this.